src/data/constants.js

```diff
@@ -0,0 +1,17 @@
+export const menuLinks = [
+  {
+    id: 1,
+    path: '/',
+    text: 'Books',
+    element: '<Books />',
+  },
+  {
+    id: 2,
+    path: '/categories',
+    text: 'Categories',
+    element: '<Categories />',
+  },
+];
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Kindly make sure each value of your action type correctly follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux) as required for the project. They should look to something like this:
```JavaScript
 const ADD_BOOK = bookstore/books/ADD_BOOK ;
 const REMOVE_BOOK = bookstore/books/REMOVE_BOOK;
  ...
```

Reply:
Ok. Thanks.